adaptive_shield/README.md

```diff
@@ -0,0 +1,17 @@
+## Overview
+SaaS Apps like Office 365, Slack, Zoom, and Salesforce are vital to a business's day-to-day operations, but can often introduce new security challenges. Adaptive Shield's SaaS Security Posture Management solution (SSPM) features proactive, deep, continuous and automated monitoring and management of all business critical SaaS applications. By integrating with all SaaS apps within minutes, Adaptive Shield enables security teams to gain complete control over their SaaS apps, allowing them to harden their security policies and reduce risks.
+
+With the Adaptive Shield integration, you can track and monitor SaaS posture alerts, such as configuration drifts, integration failures, and security check degradations, as Datadog Events.
+
+## Setup
+
+1. Click **Connect Accounts** to begin authorization of this integration. You will then be redirected to [Adaptive Shield][1].
+2. Click on **"Add New Alerts Channel"** -> Select **Datadog**.
+3. Provide the alias name.
+4. Click on **OAuth** to authorize.
```

Comment:
We'll also need to add uninstallation instructions here. Please add the following (you can just click commit suggestion). If there are any specific uninstallation instructions that users need to follow, please add those here as well. 

```suggestion
4. Click on **OAuth** to authorize.


### Uninstallation

Once you uninstall this integration, any previous authorizations will be revoked. 

Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys management page](https://app.datadoghq.com/organization-settings/api-keys).

```